docs/getting-started/quick-start.md

```diff
@@ -250,6 +276,100 @@ fun main() {
 }
 ```
 
+### IOS with SwiftUI
+
+1. Create `RootHolder` class that holds the root component and its lifecycle.
+
+```swift
+class RootHolder : ObservableObject {
+    let lifecycle: LifecycleRegistry
+    let stateKeeper: StateKeeperDispatcher
+    let root: RootComponent
+
+    init(savedState: ParcelableParcelableContainer?) {
+        lifecycle = LifecycleRegistryKt.LifecycleRegistry()
+        stateKeeper = StateKeeperDispatcherKt.StateKeeperDispatcher(savedState: savedState)
```

Comment:
Let's keep Quick Start docs simple and don't mention StateKeeper on iOS for now. Also let's remove everything related to state preservation on iOS. This stuff is highly experimental. I'm going to create a separate docs page for iOS integration with all the details.